common/models/account.js

```diff
@@ -1,8 +1,18 @@
+var ethUtil = require('ethereumjs-util');
+var sigUtil = require('eth-sig-util');
+
 module.exports = function(Account) {
-  Account.greet = function(msg, cb) {
-    process.nextTick(function() {
-      msg = msg || 'hello';
-      cb(null, 'Sender says ' + msg + ' to receiver');
-    });
-  };
+
+  Account.observe('before save', function(ctx, next) {
+    const text = "Please sign in to Alchemy";
```

Comment:
I don't think its very secure to just use this same message for everyone and have it in the codebase like this, but not sure what another easy solution would be for now

Reply:
Why isn't it secure to use the same message for all accounts?

Reply:
hmm, yeah nevermind